public/style.css

```diff
@@ -9,15 +9,43 @@ a{
     color: inherit;
 }
 
+.dbVideoList{
+    display: grid;
+    grid-template-columns: auto auto auto;
+}
```

Comment:
Consider combining if they serve the same purpose. Use descriptive names.
```
.dbVideoList, .videoList {
    display: grid;
    grid-template-columns: auto auto auto;
}

```

Reply:
okay, this will be handled.